src/ClusterBootstrap/services/logging/fluentd.yaml

```diff
@@ -1,52 +1,58 @@
-kind: DaemonSet
 apiVersion: apps/v1
+kind: DaemonSet
 metadata:
-  name: fluentd-es-v2.0.2
+  name: fluentd
   namespace: kube-system
   labels:
-    k8s-app: fluentd-es
-    version: v2.0.2
+    k8s-app: fluentd
     kubernetes.io/cluster-service: "true"
     addonmanager.kubernetes.io/mode: Reconcile
 spec:
   selector:
     matchLabels:
-      k8s-app: fluentd-es
+      k8s-app: fluentd
   template:
     metadata:
       labels:
-        k8s-app: fluentd-es
+        k8s-app: fluentd
         kubernetes.io/cluster-service: "true"
-        version: v2.0.2
-      # This annotation ensures that fluentd does not get evicted if the node
-      # supports critical pod annotation based priority scheme.
-      # Note that this does not guarantee admission on the nodes (#40573).
       annotations:
-        scheduler.alpha.kubernetes.io/critical-pod: ''
+        prometheus.io/scrape: 'true'
+        prometheus.io/path: '/metrics'
+        prometheus.io/port: '{{cnf["fluentd"]["port"]}}'
     spec:
+      hostNetwork: true
       containers:
-      - name: fluentd-es
-        image: {{cnf["dockers"]["container"]["fluentd-elasticsearch"]["fullname"]}}
+      - name: fluentd
+        image: fluent/fluentd-kubernetes-daemonset:v1.7.4-debian-elasticsearch6-1.1
```

Comment:
please change the docker image to configurable parameter. Default value can be found in params.py